### ISSUE TITLE
Placing Minefield Sign will aggro the tower if it's the only target within its attack radius. 

### ISSUE THREAD

**Issue by Ebic** [ValveSoftware/Dota2-Gameplay#9607]
### Ability name

Minefield Sign

### Description

https://www.reddit.com/r/DotA2/comments/143hbvk/building_techies_minefield_sign_will_aggro_the/?utm_source=share&utm_medium=web2x&context=3

The minefield sign will aggro towers if it is placed within their attack radius and will subsequently be destroyed. This will only occur if there is nothing else aggro'd at the same time. 

If the tower is already aggro'd on another unit, and you place the sign within its attack radius, the tower will never target the sign. 

### Example Match ID (and possibly Timestamp)

video explains it

**Comment by commentator24** [comment]
maybe it's intended?
any hero, minions can attack it too with attack ground command, not just towers. then the minefield area show but does no dmg

**Comment by Ebic** [comment]
Yeah you're right it does take damage from anything, but why? It seems to have ~10hp, so a single attack from pretty much anything will destroy it.

It seems kinda unintentional that it can be destroyed during the cast animation and still go on cooldown. It’s even more annoying now that the cast time is 2s - especially when you want to use the aghs upgrade. 

I feel it shouldn’t take damage whilst you’re casting it since it’s not like you can damage it when it’s placed anyway.

**Comment by TheUnholyWrath** [comment]
#8440
